docs/tp/10.md

```diff
@@ -0,0 +1,282 @@
+---
+sidebar_position: 10
+description:  Utilisation des fils, Utilisation des mutex et des canaux
```

Comment:
This is not correct :)

Reply:
Do you mean the `Utilisation des fils` part? Or which one?

Reply:
yes

Reply:
Maybe `fils d'execution`?